modules/ROOT/pages/tuning-caching.adoc

```diff
@@ -0,0 +1,19 @@
+= How Caching Use Helps to Improve Processing Performance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Identifying key aspects of your data such as when to use cache and which cache strategy to follow, helps you to improve processing performance:
+
+* Is the data frequently called with often repeated results?
+* Can you afford eventual consistency between the data source and the cache information? At least for a while until cache refreshes.
+
+Mule runtime engine (Mule) offers customizable strategies, such as Cache scope and HTTP Caching API Gateway policy, to enable cache according to your needs. +
+Both strategies help applications that constantly request the same data by reducing the processing load, especially network interactions with a remote host.
+
+For Cache scope, in a production environment, the default in-memory caching configuration might consume all your memory resources. To avoid this consumption, use an object-store caching configuration with well-defined storage and expiry policies.
```

Comment:
I suggest ending this topic with the previous paragraph and saving the example of the Caching scope for its appropriate subtopic.
The following text does not really tell the reader what the Cache scope is or how it solves the problem caused by in-cache memory, and it does not mention the other strategy, the policy, at all.

If you decide to keep this text anyway, clean it up as follows?
```suggestion
For example, consider a production environment in which the default in-memory caching consumes all of your memory resources. To avoid this, use object-store caching with well-defined storage and expiry policies.
```

Reply:
Again, at this point does your audience know that object-store caching = the Cache scope or what well-defined storage and expiry policies = HTTP Caching API Gateway policies are?